packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -50,6 +52,8 @@ export class PluginViewWidget extends Panel implements StatefulWidget {
         this.node.style.height = '100%';
     }
 
+    public onDescriptionChange: Emitter<void> = new Emitter<void>();
```

Comment:
Can be removed, no need to duplicate the event and instead use the widget directly.

Reply:
As mentioned above, the wrapped widget actually doesn't have the `DescriptionWidget` interface implemented, but `plugin-view-widget` does so is needed